ansible/roles/install-basecoin/files/key.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
These are generated but static files. They should be part of the repository instead of being generated. I think I have a case open with it somewhere.